pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -389,8 +389,19 @@ type OIDCSettings struct {
 	ExtraScopes   string `json:"extraScopes,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=metadata;recommended;minimal
```

Comment:
please rerun `./hack/update-codegen.sh` and also commit modified CRD files (in charts/kubermatic-operator), and don't forge to increment chart version in charts/kubermatic-operator/Chart.yaml

Reply:
and `./hack/update-swagger.sh` too